src/components/select/filterable-select/filterable-select.test.tsx

```diff
@@ -699,6 +729,25 @@ describe("when the input is focused", () => {
     expect(screen.getByRole("combobox")).toHaveValue("red");
   });
 
+  it("if the values are objects, deleting the last charcter with backspace works as expected and the correct match text is provided", async () => {
```

Comment:
**nitpick:** Just a minor grammatical/typo improvement suggestion.

```suggestion
  it("should delete the last character with backspace and the correct match text is provided, if the values are objects", async () => {
```

Reply:
Refactored the test description, as per the suggested improvement.